### ISSUE TITLE
Add a timeline with the ability to export to ics format in a more prominent place

### ISSUE THREAD

**Issue by robbyoconnor** [outreachy/website#374]
I would like one uniform place to look for the timeline. We just have to handle the current round, though having both rounds is also handy. This should be prominent similar to how Google does it for GSoC: 

![image](https://user-images.githubusercontent.com/23088/76821102-5df81d00-67e3-11ea-8dcf-587f85eeef6c.png)

Being able to export it is also super helpful since i'm running both Outreachy and GSoC so keeping dates straight is handy so I can nudge people to do things.

**Comment by wetneb** [comment]
I agree it would be very useful to have the entire timeline for a round in a single place.

There is https://github.com/outreachy/website/issues/467 which is about adding an ICS export. Which also lists how dates are currently scattered across various pages.
